includes/class-modal-checkout.php

```diff
@@ -1527,6 +1527,15 @@ public static function get_subscription_limited_message() {
 		);
 	}
 
+	/**
+	 * Alternative error message to show when limiting a subscription product's purchase to one of any status.
+	 *
+	 * @return string
+	 */
+	public static function get_subscription_limited_message_any() {
+		return __( "You're already a subscriber! You can only have one subscription at a time. If you wish to renew an expired subscription, please Sign In and visit the subscriptions page on your account.", 'newspack-blocks' );
```

Comment:
Nit:

```suggestion
		return __( "You're already a subscriber! You can only have one subscription at a time. If you wish to renew an expired subscription, please sign in and visit the subscriptions page on your account.", 'newspack-blocks' );
```

It would also be great if we could omit the "Could not complete this transaction. Please contact us for assistance" part of the message in this scenario, but I'd consider that a bonus.